### ISSUE TITLE
Stops working when trying to save

### ISSUE THREAD

**Issue by SirDagen** [microsoft/vscode#73366]
Issue Type: <b>Bug</b>

After a certain time this programm stops working when I try to save. It just freezes. And cannot be reopened. If I reopen it, it still is frozen ("another instance of 'code' is running"). 

The file is on an server. (I can still open and save the file with my text editor "xed".)

I have to restart my computer to get it working again. 

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:20:22.771Z)
OS version: Linux x64 4.18.0-17-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 2700)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>surface_synchronization: enabled_on<br>video_decode: unavailable_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 0|
|Memory (System)|23.44GB (20.57GB free)|
|Process Argv|--unity-launch|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
numbered-bookmarks|ale|6.2.0
gitlens|eam|9.7.1
python|ms-|2019.4.12954
vsliveshare|ms-|1.0.125
quicktype|qui|12.0.46

</details>

**Comment by chrmarti** [comment]
What type of filesystem is this?

**Comment by SirDagen** [comment]
Hello, thank you for answering. The company told me they use XFS.

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by SirDagen** [comment]
I do believe that my problem has to do something with the bug in the filemanager I found and reported ( https://github.com/linuxmint/nemo/issues/2133 ), where the timestamp of a saved file is always one hour ahead (into the future). This might have caused the problems I described above when trying to save my file.  

Interestingly this timestamp thing only happens with a FTP connection, but *NOT* if I connect via SFTP ( as I found out). 

The reason why I assume this link between my problems and the wrong functioning of the write timestamp is, that since I use SFTP for the connection to my server, all problems with vscode have disappeared. - However I am quite sure, if I return to the FTP connection I am going to experience the problems again.
